packages/editor/src/components/parts/common/table/useResizableEditableTable.tsx

```diff
@@ -61,12 +61,10 @@ const useResizableEditableTable = <TData,>({
   useEffect(() => {
```

Comment:
In general I think this was the real problem.
However I hadn't the currage to change this to something else and break some UI, so I fixed it in the MailAttachmentPart directly.